Ellabit/Challenges/Challenge151Removeduplicatesfromanarray.cs

```diff
@@ -81,9 +81,9 @@ public class TestChallenge
 
 notes tests contain arrays with""";
         public List<string> Tests { get; set; } = new string[] { "Test1", "Test2", "Test3" }.ToList();
-        private Dictionary<string, string> _tags = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
+        private Dictionary<string, string> _tags = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase) { { "Array", "Duplicates"}, { "Level","2"} };
         public Dictionary<string, string> Tags { get => _tags; set => _tags = value; }
-        public bool ShowBlockly { get => false; }
```

Comment:
IronBlock's Blockly doesn't have a way to return an array. Thus I would set Blockly = false